### ISSUE TITLE
[1.12.2] Bundles and relies on outdated Kotlin

### ISSUE THREAD

**Issue by LlamaLad7** [TeamWizardry/LibrarianLib#176]
This mod relies on and bundles Kotlin 1.3.0, and makes use of experimental features at the time of development (unsigned types), and so when other mods bundle Kotlin 1.5.30, this mod breaks due to the unsigned types methods having changed.
It would be great if you could compile against and bundle a more recent version.

**Comment by malusafada** [comment]
a

**Comment by thecodewarrior** [comment]
I'm currently working on trying to jury-rig together a new build, but I need to update gradle to update kotlin, which means I need to update ForgeGradle, which is a nightmare, so it may be a bit.

**Comment by SystemlessDev** [comment]
Any progress on this?

**Comment by thecodewarrior** [comment]
Unfortunately not. Work has been stressful lately, so I've had very little motivation for working on personal projects. Especially stressful and mentally taxing projects like this one.

**Comment by SystemlessDev** [comment]
Gotcha. Prioritize work and irl over a Minecraft mod. It's way more important with your mental health and all that. Hope things become better soon ❤️😁

**Comment by ChAoSUnItY** [comment]
There's currently an updated version of Forgelin calls Forgelin Continuous (I'm the author of it), and there's no additional content compared to original one, so you may probably want to give it a try.

Link: https://www.curseforge.com/minecraft/mc-mods/forgelin-continuous

**Comment by thecodewarrior** [comment]
Yeah, that was my plan, but in order to compile against newer kotlin stdlib versions I have to use newer versions of gradle, which requires upgrading forgegradle, which is a nightmare.
